README.md

```diff
@@ -1,70 +1,90 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
- [ ] I couldn't find the link for the backend.  For your next review request, kindly include the link to the open PR on your backend project.